magento/magento2
# MAGETWO-54158

### Description

Added an acl.xml to the send-friend module
Added upgrade data script for backward compatibility for ACL-roles that were already selected for users.

Changed the acl.xml for the backend module
Added upgrade data script for backward compatibility for ACL-roles that were already selected for users.

### Fixed Issues (if relevant)

1. magento/magento2#MAGETWO-54158: ACL not defined for Magento_Backend::web

### Manual testing scenarios

1. Install Magento latest develop version
2. Go to Admin side
3. Click "System > User Roles"
4. Click "Add new Role" enter Role name
5. In left side choose Role Resources and check "Dashboard", "Web Section", "Email to a friend", "Currency Setup", "Store E-mail addresses"
6. Save Role
7. Click "System > All Users"
8. Click "Add new User"
9. Enter data in required fields
10. In left side click "User Role" and choose early created role.
11. Click "Save"
12. Sign out and Log in using credentials of new user
13. Click "Stores > Configuration"

Expected result:
All the tabs that were selected in the role should be visible

### Contribution checklist
